django/contrib/auth/admin.py

```diff
@@ -205,6 +205,7 @@ def response_add(self, request, obj, post_url_continue=None):
         # * The user has pressed the 'Save and add another' button
         # * We are adding a user in a popup
         if '_addanother' not in request.POST and IS_POPUP_VAR not in request.POST:
+            request.POST = request.POST.copy()
```

Comment:
I thought of passing post_url_continue None or url, instead of messing with request.POST
and in the base method checking if default of post_url_continue is overridden.
But then I felt this is not the best approach when dealing with methods.